### PR TITLE
update field stars USNO

### PR DESCRIPTION
**Pull request by dave-kaye**
This will update the help files for the field stars section USNO.  It is independent of any code or update to catalog files.

### CODE REVIEW

File: GUI/xephem/help/xephem.html [hunk 1]
```diff
@@ -8102,39 +8102,36 @@ <h3><a class="mozTocH3" name="mozTocId676533"></a> 7.5.2 <span
...
+Monet, A., Rhodes, A., Riepe, B., Sell, S., Stone, R., Vrba, F., Walker, R.,
+1998, USNO-SA2.0, (U.S. Naval Observatory, Washington DC). </div>
 <br>
 This catalog has been included with permission of USNO as long as
 we mention the following stipulations:<br>
 <br>
 <div style="margin-left: 40px;"> It may not be the latest version,
-check
-with http://ad.usno.navy.mil. </div>
+check with http://www.usno.navy.mil. </div>
```
Comment (brandon-rhodes) [inline GUI/xephem/help/xephem.html:8127]: Do you think we could be more specific with the URL? That top-level page will require a bit of searching for users unfamiliar with navigating the USNO web site. If we used something like:

https://www.usno.navy.mil/USNO/astrometry/optical-IR-prod

— then they might find catalog information more readily?

File: GUI/xephem/help/xephem.html [hunk 2]
```diff
@@ -8102,39 +8102,36 @@ <h3><a class="mozTocH3" name="mozTocId676533"></a> 7.5.2 <span
...
 the real sky, but it is great for its intended use as an astrometric
-mesh for comet hunters or such. To order these catalogs, see <a
- href="http://psyche.usno.navy.mil/pmm" target="new">http://psyche.usno.navy.mil/pmm</a>.
-If you have such a catalog, simply enter the name of its base directory
-and toggle this switch on. The default assumes a symbolic link,
-"catalogs/usno" off the shared directory. The suggested citation for
-SA1.0 follows:<br>
+mesh for comet hunters or such. If you have such a catalog, simply
+enter the name of its base directory and toggle this switch on.
+The default assumes a symbolic link, "catalogs/usno" off the shared
```
Comment (brandon-rhodes) [inline GUI/xephem/help/xephem.html:8112]: I haven't checked the code — does it really require a symbolic link? Or is it just as happy if the catalog has been actually moved to that disk location?

**Review by brandon-rhodes — COMMENTED** [review]
Thanks for thinking about updating this old documentation! I've made one or two comments. There are further tweaks that I'll probably make (to a few commas in particular), but I can make those once this is landed; there's no need to bother with them before landing. Thanks!

**Comment by dave-kaye** [comment]
Does not require a "symbolic" link.  Directory is fine.  I checked for full and SA catalog versions in the called out directory before submitting, and they both worked.  Just Downey's verbage, did not change it.  He may have used a link at some point in history?

 

As to link, do as you wish.  I kept it high, because whenever I went through that area, I kept finding notices that they are updating the entire site - to be completed months ago.  I figured the site will still be there, but the locations within the site might change.  That way it would still be valid.

 
 

Sent: Friday, October 29, 2021 at 6:16 PM

**Comment by dave-kaye** [comment]
Please ignore the commit on Nov 2.  I was uploading to a branch and got an error message, it must have moved it to where I did not want it.  Finding out that if there is an error, it drops to main branch and if you retry it loads it there. It is a png that is part of another request that is awaiting completion of this one.  Can use only the web interface, so am limited on what I can do, including no rollbacks.

**Comment by dave-kaye** [comment]
Github problem on push led to file being put in wrong branch.  If you see a png file from Nov 2, please ignore/delete.   I think I got it removed.  It is part of a second request that is awaiting completion of this one.  Newbie and web interface is limited....

**Comment by brandon-rhodes** [comment]
Thanks for the additional information! With the reformatting, I'd missed that the previous text said "symbolic link", so that can be tackled separately; and once this lands I'll probably make the USNO link more specific. I can always change it later as the content is chased around their site by redesigns.

I don't see any image file in the current diff, so I think you got it out of this PR successfully!

I'll plan to merge this in the morning, when my proofreading skills are at their height :)

**Pull request merged**